pyproject.toml

```diff
@@ -7,7 +7,7 @@ requires = [
 ]
 
 [project]
-name = "xrtpy"
+name = "XRTpy"
```

Comment:
Not sure if this is correct.

Reply:
I do have a tendency of keeping/using it capitalized

Reply:
Does this mean that one would have to do `pip install XRTpy` on case sensitive OSes?

Reply:
No for pip, this is case insensitive.